embedded/remotestorage/s3/s3.go

```diff
@@ -118,7 +128,15 @@ func Open(
 				return http.ErrUseLastResponse
 			},
 		},
-	}, nil
+		s3CredentialsRefreshTicker: time.NewTicker(time.Minute),
```

Comment:
It would be better to have this ticker created only if needed (i.e., when a s3 role is assigned)